src/Rules/PropertyConditions/SystemProperties.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Axe.Windows.Rules.PropertyConditions
+{
+    static class SystemProperties
+    {
+        private static readonly string SystemISOLanguageName = System.Globalization.CultureInfo.CurrentCulture.ThreeLetterISOLanguageName;
+
+        internal static string OverriddenISOLanguageName { get; set; }
+
+        public static Condition IsEnglish = CreateEnglishConditionWithTestOverride();
+
+        private static Condition CreateEnglishConditionWithTestOverride()
+        {
+            StringProperty cultureName = new StringProperty(_ => (OverriddenISOLanguageName ?? SystemISOLanguageName));
+
+            return cultureName.IsNoCase("eng");
+        }
+    } // class
```

Comment:
What is the purpose of these comments? Should they also be placed at the end of functions (or are classes/namespaces a special case)? Why is indentation insufficient to determine what the braces close?

Reply:
This is @RobGallo's pattern. His text editor isn't particularly good about indents, so he uses this technique to help clarify where classes end. It's probably more valuable in cases where we have multiple classes in a source file, which is pretty rare in our code. I started the class by copying and pasting another class, so this came along for the ride.

I find 349 instances of `// namespace` and 331 instances of `// class`. If we want to clean those up, it feels like a holistic change to consider